app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,17 @@
+class AccountActivationsController < ApplicationController
+  def edit
+    user = User.find_by(email: params[:email])
+    
+    if user&.unactivated? && user.authenticated?(:activation, params[:id])
```

Comment:
I think you'll need to add `user&.authenticated` here too

Reply:
The default condition was `if user && user.unactivated? && user.authenticated?`. 
I tried to use `&.` operator for both conditions with something like `if user&.unactivated?&.authenticated?`  but I found out that this condition will be translated to `if user && user.unactivated? && user.unactivated.authenticated?` and this is not the desired behavior.
Also, I used `user&.unactivated? && user&.authenticated?` and it double-checked `user`, so I considered it’s better to avoid this.
What should I do?